wails/wails_handlers.go

```diff
@@ -374,7 +374,7 @@ func (app *WailsApp) WailsRequestRouter(route string, method string, body string
 				"route":    route,
 				"method":   method,
 				"body":     body,
-				"currency": "usd",
+				"currency": app.svc.GetConfig().GetCurrency(),
```

Comment:
is this necessary?

Reply:
if we fail to get bitcoin rate logs shall show for which currency it failed. no?

Reply:
Why is it only logged in the wails handler? I don't see this in the http version despite it being much more used.

Also, I think it's wrong to log the currency at this level. I think it's better to include that in the errors logged at `GetBitcoinRate` in the OAuth service

Reply:
resolved